kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -1,24 +1,82 @@
 <template>
 
-  <div class="roster">
-    <div>
-      <user-create-modal></user-create-modal>
-    </div>
-    <div class="user-roster">
-      <ul>
-        <li v-for="user in getUsers">
-          <div>{{ user.last_name + ", " + user.first_name + ", "}} {{user.roles.length ? user.roles[0].kind : "user" }}</div>
-          <user-edit-modal :userid="user.id" :roles="user.roles" :username="user.username" :firstname="user.first_name" :lastname="user.last_name"></user-edit-modal>
-          <button @click="deleteUser(user.id)">Delete</button>
-        </li>
-      </ul>
-    </div>
-  </div>
-  <div class="sidebar">
-    <div class="user-count">
-      <div>Total:</div>
-      <div>{{ getUsers.length }}</div>
+  <div class="user-roster">
+
+    <h2>
+      All Users
+    </h2>
+
+    <span> ( {{ users.length }} )</span>
+
+    <div class="toolbar">
+
+      <select v-model="roleFilter" name="user-filter">
+        <option selected value="all"> All Users </option>
+        <option v-bind:value="role" v-for="role in roles">
+          {{role.kind | capitalize}}s
```

Comment:
We can't do this because it cannot be translated. Can you update this to a hard-coded list of possible roles and their display names?